src/App.vue

```diff
@@ -2,37 +2,91 @@
   <v-app>
     <v-toolbar app>
       <v-toolbar-title class="headline text-uppercase">
-        <span>Vuetify</span>
-        <span class="font-weight-light">MATERIAL DESIGN</span>
+        <span>Posts</span>
       </v-toolbar-title>
       <v-spacer />
-      <v-btn
-        flat
-        href="https://github.com/vuetifyjs/vuetify/releases/latest"
-        target="_blank"
-      >
-        <span class="mr-2">Latest Release</span>
-      </v-btn>
     </v-toolbar>
-
     <v-content>
-      <HelloWorld />
+      <v-card min-height="100%">
+        <v-container
+          grid-list-lg
+        >
+          <v-layout>
+            <v-flex>
+              <v-text-field
+                v-model="searchText"
+                label="Post Id"
+                single-line
+                outline
+              />           
+            </v-flex>
+            <v-flex> 
+              <v-btn
+                large 
+                color="primary"
+                @click="loadPost"
+              >
+                Load
+              </v-btn>
+            </v-flex>
+          </v-layout>
+          <post
+            v-for="(post, index) in posts"
```

Comment:
I think there will only be one post returned for an ID so a loop won't be required here.